tests/api2/test_026_kubernetes_backup_chart_releases.py

```diff
@@ -239,3 +241,78 @@ def test_24_get_k3s_logs():
         ks3_logs = open(f'{artifacts}/k3s-scale.log', 'w')
         ks3_logs.writelines(results['output'])
         ks3_logs.close()
+
+    @pytest.mark.dependency(name='truecharts_calalog')
+    def test_25_set_truechart_catalog():
+        global payload, results
+        payload = {
+            'force': True,
+            'preferred_trains': ['stable'],
+            'label': 'TRUECHARTS',
+            'repository': 'https://github.com/truecharts/catalog.git',
+            'branch': 'main'
+        }
+        job = call('catalog.create', payload)
+        job_status = wait_on_job(job, 600)
+        assert job_status['state'] == 'SUCCESS', str(job_status['results'])
+
+    @pytest.mark.dependency(name='release_truechart_plex')
+    def test_26_create_plex_chart_release(request):
```

Comment:
Instead of creating separate tests for these as they are not really what we want to test, i believe you should use some sort of try/finally or context manager approach where you can setup the pre-reqs before validating the structure and then after you have done the tests ( regardless of failure ) cleanup on exist.